lib/enum/arrays.rb

```diff
@@ -1,24 +1,86 @@
 class Array
+  def iniinitialize
+    @test_array
+  end
 
   def sum_of_numeric_elements
-  end
+    self.map{|num|
```

Comment:
This works, but could be made a lot shorter. Anytime you have a map method with an if-statement inside, you can probably use a select method instead. Select will evaluate the block for each element, and then return just the elements where the block evaluates to true. 

In this case, you could use select to get the numeric elements, and then sum them the same way you had it: 

`self.select { |num| num.is_a? Numeric }.reduce(:+)`